test/chernoff.jl

```diff
@@ -148,11 +148,114 @@
         2.000 0.0001212
     ]
 
-    for i=1:size(cdftest, 1)
+    for i=1:size(cdftest,1)
         @test isapprox(cdf(d, cdftest[i, 1]), cdftest[i, 2]) 
     end
     
     for i=1:size(pdftest, 1)
         @test isapprox(pdf(d, pdftest[i, 1]), pdftest[i, 2] ; atol = 1e-6)
     end
+
+    # test quantiles
+    qs = [0.5,
```

Comment:
They are from:
Groeneboom, Piet; Wellner, Jon A. (2001). "Computing Chernoff's Distribution". Journal of Computational and Graphical Statistics

Reply:
Ah, great! Could you add a comment to make it clear that these are taken from the literature and not just computed with Distributions.jl?

Reply:
> Ah, great! Could you add a comment to make it clear that these are taken from the literature and not just computed with Distributions.jl?

sure,



> And I assume the tests fail on the master branch?

I cannot see the CI output to examine it!

Reply:
> I cannot see the CI output to examine it!

You would have to run the test locally with the latest release of Distributions. But I just checked and indeed it fails since `newton` is not defined :+1: